eng/packages/http-client-csharp/generator/TestProjects/Local/Mgmt-TypeSpec/src/MgmtTypeSpec.csproj

```diff
@@ -10,14 +10,33 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="System.ClientModel" Version="1.1.0-beta.4" />
-    <PackageReference Include="System.Text.Json" Version="8.0.5" />
-		<PackageReference Include="Azure.Core" Version="1.43.0" />
+    <PackageReference Include="System.ClientModel" />
+    <PackageReference Include="System.Text.Json" />
+	  <PackageReference Include="Azure.Core" />
+	  <PackageReference Include="Azure.ResourceManager" />
   </ItemGroup>
 	
 	<!-- Include shared code from Azure.Core -->
 	<ItemGroup>
 		<Compile Include="$(MSBuildThisFileDirectory)..\..\..\..\..\..\..\..\sdk\core\Azure.Core\src\Shared\AzureKeyCredentialPolicy.cs" LinkBase="Shared/Core" />
 		<Compile Include="$(MSBuildThisFileDirectory)..\..\..\..\..\..\..\..\sdk\core\Azure.Core\src\Shared\RawRequestUriBuilder.cs" LinkBase="Shared/Core" />
+		<Compile Include="$(MSBuildThisFileDirectory)..\..\..\..\..\..\..\..\sdk\core\Azure.Core\src\Shared\OperationInternal.cs" LinkBase="Shared/Core" />
```

Comment:
Is this being manually modified? I think it should be part of NewAzureProjectScaffolding under some condition that indicates if the library is using LRO.

Reply:
updated